airflow/www/views.py

```diff
@@ -3127,6 +3127,44 @@ def robots(self):
         """
         return send_from_directory(current_app.static_folder, 'robots.txt')
 
+    @expose('/audit_log')
+    @auth.has_access(
+        [
+            (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG),
```

Comment:
```suggestion
            (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG),
            (permissions.ACTION_CAN_READ, permissions.RESOURCE_AUDIT_LOG),
```

Even though a lot of historical information is also present in the DAG view, I think we should prevent users from viewing this page unless they have full audit log access.